ThirdPartyNotices.txt

```diff
@@ -0,0 +1,1358 @@
+THIRD PARTY SOFTWARE NOTICES AND INFORMATION
+
+Do Not Translate or Localize
+
+This software incorporates material from third parties.
+Microsoft makes certain open source code available at https://3rdpartysource.microsoft.com,
+or you may send a check or money order for US $5.00, including the product name,
+the open source component name, platform, and version number, to:
+
+Source Code Compliance Team
+Microsoft Corporation
+One Microsoft Way
+Redmond, WA 98052
+USA
+
+Notwithstanding any other terms, you may reverse engineer this software to the extent
+required to debug changes to any libraries licensed under the GNU Lesser General Public License.
+
+---------------------------------------------------------
+
+GoogleTestAdapter 0.18.0 - Apache-2.0
+
+
+
+Copyright 2015-2019 Christian Soltenborn
+
+Licensed under the Apache License, Version 2.0 (the "License"); you may not use this software except in compliance with the License. You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+
+---------------------------------------------------------
+
+---------------------------------------------------------
+
+monotonic 1.6 - Apache-2.0
```

Comment:
what's this library? I think we should generate this file based on C++ third party libraries.

Reply:
I am unsure, the whole file was generated by Component Governance on ADO for our repository. Do you know how it can be generated based on C++ third party libraries? Or do you mean to generate it manually?

Reply:
You can check how ONNXRuntime generates the file, since the content looks like making more sense.
https://github.com/microsoft/onnxruntime/blob/main/ThirdPartyNotices.txt